plugins/genesis-tools/commands/github-pr.md

```
@@ -383,16 +408,20 @@ When the user provides **multiple PR URLs/numbers**, switch to analysis-first mo
 ```bash
 date -u +"%Y-%m-%dT%H-%M-%S"   # e.g. 2026-02-18T23-18-22
 ```
 
-Plan files go to: `.claude/plans/reviews/PR-<id>-<datetime>.md`
+**Save location depends on `--save` flag:**
+- Without `--save`: plan files go to `/tmp/github/reviews/<repo>/PR-<id>-<datetime>.md`
+- With `--save`: plan files go to `.claude/plans/reviews/PR-<id>-<datetime>.md`
+- With `--save <path>`: plan files go to `<path>/PR-<id>-<datetime>.md` (relative to original cwd)
+
+Create the target directory:
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)